raspberry-pi/4/tc358743.nix

```
@@ -0,0 +1,211 @@
+{ config, lib, ... }:
+
+let
+  cfg = config.hardware.raspberry-pi."4".tc358743;
+in
+{
+  options.hardware = {
+    raspberry-pi."4".tc358743 = {
+      enable = lib.mkEnableOption ''
+        Enable support for the Toshiba TC358743 HDMI-to-CSI-2 converter.
+
+        This can be tested with a plugged in converter device and for example
+        running ustreamer (which starts webservice providing a camera stream):
+        ${pkgs.ustreamer}/bin/ustreamer --persistent --dv-timings
```

Escape this package path; including package paths in options can break things when generating documentation.

```suggestion
        ''${pkgs.ustreamer}/bin/ustreamer --persistent --dv-timings
```